File: modul3/Python/PRAK304_2310817210017_MUHAMMADNURHUDABAIHAQI.py
while True:
    nilai = int(input(""))

    if nilai == 0:
        print("nol")
    elif nilai > 0 and nilai <=9:
        print("Satuan")
    elif nilai == 10:
        print("Puluhan")
    elif nilai >= 11 and nilai <=19:
        print("Belasan")
    elif nilai >= 20 and nilai <=99:
        print("Puluhan")
    else :
        print("Anda Menginput Melebihi Limit Bilangan")

    

   

File: modul3/C/PRAK304_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int nilai;

    while (1) {
        printf("");
        scanf("%d", &nilai);


        if (nilai == 0) {
            printf("Nol\n");
        } else if (nilai >= 1 && nilai <=9) {
            printf("Satuan\n");
        } else if (nilai == 10) {
            printf("Puluhan\n");
        } else if (nilai >= 11 && nilai <=19) {
            printf("Belasan\n");
        } else if (nilai >= 20 && nilai <=99) {
            printf("Puluhan\n");
        } else {
            printf("Anda Menginput Melebihi Limit Bilangan\n");
        }
    
    }

    return 0;
}
